containers/payments/PaymentVerificationModal.js

```diff
@@ -84,6 +88,12 @@ const PaymentVerificationModal = ({
         }
     };
 
+    useEffect(() => {
```

Comment:
Put this in the useState up there

```suggestion
const [step, setStep] = useState(() => isDuckDuckGo() ? STEPS. DUCKDUCKGO  : STEPS.REDIRECT);
```